spec/api/payment/void_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Afterpay::API::Payment::Void do
+  describe '#call' do
+    let(:token) { 'some_token' }
```

Comment:
You don't need the token in the void request, only the `amount` https://developers.afterpay.com/afterpay-online/reference#void-payment

You can use [this spec](https://github.com/nebulab/afterpay-ruby/blob/main/spec/api/payment/refund_spec.rb) as a reference

Reply:
Removed the token as it is only needed in the Auth